src/formatting/expr.rs

```diff
@@ -496,7 +505,12 @@ fn block_prefix(context: &RewriteContext<'_>, block: &ast::Block, shape: Shape)
                     )?
                 )
             } else {
-                "unsafe ".to_owned()
+                match context.config.control_brace_style() {
+                    ControlBraceStyle::AlwaysNextLine => {
+                        format!("unsafe\n{}", shape.indent.to_string(&context.config))
```

Comment:
Could just use `to_string_with_newline` here instead since the newline isn't going to be conditional in this case
```suggestion
                        format!("unsafe{}", shape.indent.to_string_with_newline(&context.config))
```

Reply:
Will do